compiler/src/iree/compiler/Codegen/Common/CombineLayoutTransformation.cpp

```diff
@@ -42,6 +43,9 @@ static void simplifyComplexRelayoutOps(RewriterBase &rewriter,
   SmallVector<linalg::PackOp> packOps(
       funcOp.getFunctionBody().getOps<linalg::PackOp>());
   for (auto packOp : packOps) {
+    if (getLoweringConfig(packOp)) {
+      continue;
+    }
```

Comment:
Honestly, I'm not sure if it is good because CPU relies on this as the fallback. What it really needs is walking through all the ops between `store_to_buffer` and `scf.forall/for`, performs the rewrite and folds it into map_scatter. I think it also holds in GPU path?

For CPU side, we should be in a better state that can disable the lowering config propagation for some cases. However we should have a decent fallback path without compilation error. Decent perf is better than compilation failure, IMO.